Makefile

```diff
@@ -472,6 +479,11 @@ edpm_deploy: input edpm_deploy_prep ## installs the dataplane instance using kus
 	$(eval $(call vars,$@,dataplane))
 	oc kustomize ${DEPLOY_DIR} | oc apply -f -
 
+.PHONY: edpm_register_dns
+edpm_register_dns: dns_deploy_prep ## register edpm nodes in dns as dnsdata
+	$(eval $(call vars,$@,infra))
+	oc apply -f ${DEPLOY_DIR}/network_v1beta1_dnsdata.yaml # TODO (mschuppert): register edpm nodes in DNS can be removed after full IPAM integration
```

Comment:
Could the hardcode `ctlplane` values in this sample be a problem if the user changes the `DATAPLANE_COMPUTE_IP` or `DATAPLANE_COMPUTE_1_IP` variables?

Reply:
yes, if someone changes the default IPs for the edpm nodes, they'd have to be registered manually via a dnsdata CR. running this target should only be a temporary until its integrated and we can remove it here.

Reply:
@fultonj  the IPAM work was merged, so it might change